data/strings/strings.txt

```diff
@@ -29411,99 +29411,97 @@
     zh-Hans = 下坡
     zh-Hant = 下坡
 
-  [elevation_profile_minaltitude]
+  [elevation_profile_min_elevation]
     tags = android,ios
-    en = Min. altitude
+    en = Min. elevation
     af = Min. hoogte
-    ar = الحد الأدنى للارتفاع
+    ar = أقل ارتفاع
     az = Min. hündürlük
     be = Мін. вышыня
-    bg = Мин. надморска височина
-    ca = Altitud mín.
-    cs = Min. nadmořská výška
+    bg = Мин. височина
+    ca = Min. elevació
+    cs = Min. výška
     da = Min. højde
     de = Min. Höhe
     el = Ελάχ. υψόμετρο
-    es = Altura mínima
-    es-MX = Altura mínima
+    es = Min. elevación
+    es-MX = Min. elevación
     et = Min. kõrgus
-    eu = Gutxieneko altuera
-    fa = کمینه ارتفاع
+    eu = Min. altuera
```

Comment:
Goragune minimoa

Reply:
@andergd it is possible to use some shortened version here?

Because this string should be one lined
![image](https://github.com/user-attachments/assets/fd7b5cf3-aa9f-4858-814c-3e57f1291c6f)

Reply:
Goragune min.